resources/views/tasks/show.blade.php

```diff
@@ -2,9 +2,19 @@
 
 @section('content')
     <task :task="{{ $task }}"
+          @if(auth()->user()->is_admin ==  1)
```

Comment:
Please refactor this now before it gets more complicated in the future. Use boolean values